### ISSUE TITLE
Problem with latest version

### ISSUE THREAD

**Issue by idealsceneprod** [ChordPro/chordpro#495]
## Describe the bug
The PDF preview is not showing the cords inline, but is showing them at the end of the line, highlighted in yellow.

### To Reproduce
1. Open up a .crd file
2. Click on Preview
3. Interface complains about Invalid Chord definition, although it matches what I see online
4. PDF shows words underlined, and chords at the end of each line in yellow

{define: E frets 0 7 9 9 0 0 fingers x 1 3 2 x x }
invalid chord definition: E frets 0 7 9 9 >>> 0 <<< 0 finfers x 1 3 2 x x

### Attachments
[Miracle of Sound - Valhalla Calling.pdf](https://github.com/user-attachments/files/18243404/Miracle.of.Sound.-.Valhalla.Calling.pdf)

[Miracle of Sound - Valhalla Calling.md](https://github.com/user-attachments/files/18243410/Miracle.of.Sound.-.Valhalla.Calling.md)

![image](https://github.com/user-attachments/assets/fd316043-0622-44e7-89fb-6818644fd641)

## System information
 - OS: Ubuntu 22.04
 - Where did you get ChordPro from? CPAN
Run-time information:
  ChordPro core          6.070     
  Perl                   v5.34.0    (/usr/bin/perl)
  Resource path          /usr/local/share/perl/5.34.0/ChordPro/res
  ABC support            QuickJS_XS (ABC2SVG version v1.22.18_9b12853f66 of 2024-07-08)

Modules and libraries:
  Storable               3.23      
  Object::Pad            0.813     
  Text::Layout           0.038     
  HarfBuzz::Shaper       0.027     
    library              2.7.4     
  File::LoadLines        1.046     
  PDF::API2              2.047     
  PDF::API2::XS          1.002     
  SVGPDF                 0.090     
  Font::TTF              1.06      
  JavaScript::QuickJS    0.21      
  JSON::Relaxed          0.097     
  JSON::XS               4.03

**Comment by sciurius** [comment]
Did you (by accident?) select the Modern2 preset?

**Comment by idealsceneprod** [comment]
Yes. I have most of them selected. I didn't realize those were output configurations. Thanks so much.
